compiler/rustc_expand/src/mbe.rs

```diff
@@ -72,7 +72,7 @@ enum TokenTree {
     /// A kleene-style repetition sequence, e.g. `$($e:expr)*` (RHS) or `$($e),*` (LHS).
     Sequence(DelimSpan, SequenceRepetition),
     /// e.g., `$var`.
-    MetaVar(Span, Ident),
+    MetaVar(Span, Ident, bool),
```

Comment:
The comment should explain the meaning of the `bool`. Or a two-value enum could be used.

Reply:
https://github.com/rust-lang/rust/pull/119204#discussion_r1434587296 was addressed, so this flag is no longer used.